kubernetes/admission/example_fluxinterval.rego

```diff
@@ -0,0 +1,33 @@
+# This example demonstrates how to ensure consumers of your kubernetes cluster don't slam
+# your cluster's API or git repositories by configuring [flux](https://fluxcd.io/) to sync too frequently.
+package library.kubernetes.validating.fluxinterval
+
+image := "fluxcd/flux"
+
+deny[msg] {
+	# Ensure only applies to flux images
+	containerImage := input.spec.template.spec.containers[i].image
+	contains(containerImage, image)
+
+	# Check if git poll interval arg is present
+	args := input.spec.template.spec.containers[_].args
+	contains(args[_], "--git-poll-interval")
+
+	# If single digit is present, deny
+	regex.match("--git-poll-interval=[0-9]m", args[_])
+	msg := "--git-poll-interval must be at least 10m"
+}
+
```

Comment:
I think you've got some unintended iteration happening there.  Every time you use `_` you are starting a new, nested iteration.  I put together what I think you want here: https://play.openpolicyagent.org/p/YA9ULp0WCR.  It's untested, but your tests should work.

Reply:
Thanks for the tips, I'm just learning rego syntax :)

Reply:
Looking much better.  I'd say just add the test I suggested.

Reply:
Added 👍